docs/process/labels.md

```diff
@@ -46,80 +53,75 @@ get involved with.
 | [:mag_right:][good first issue] | `good first issue` | Issues marked as ideal for a brand new contributor to start with |
 | [:mag_right:][help wanted]      | `help wanted`      | Issues marked as ideal for external contributors |
 
-### Planning
+## Collaboration
 
-We use these labels to track tasks outside the usual flow of addressing bugs or
-implementing features:
+We use these labels to track tasks that represent specific interactions as part of a productive cross-functional team:
 
-|                                   | Label name           |  Description |
-| --------------------------------- | -------------------- |  ----------- |
-| [:mag_right:][meta]               | `meta`               | Issues used to co-ordinate tasks or discuss a feature before the required work is captured |
```

Comment:
Looking at what is currently assigned to `meta`, I believe we can replace all of these with a specific `area` label that better suits what the work is. And if there's not an `area` that suits it, we can add one easily enough.